user_login.php

```diff
@@ -30,7 +30,7 @@
 include('header.inc');
 ?>
 
-<div class="bg" style='height:100%;width:100%;min-height:800px;'>
+<div class="bg valign-wrapper" style='height:100%;width:100%;min-height:800px;'>
```

Comment:
I think we would need to make the boxes and text darker if we wanted to do this semi-transparency thing
![screen shot 2018-06-28 at 10 37 34 pm](https://user-images.githubusercontent.com/2419351/42073435-f583a1fc-7b23-11e8-8cf5-9d1dd2987d53.png)

Reply:
Whoops, this is for the other branch.